test/tools/libtesteth/ImportTest.cpp

```diff
@@ -93,6 +93,7 @@ bytes ImportTest::executeTest()
 				eth::Network network = networks[j];
 				std::pair<eth::State, ImportTest::execOutput> out = executeTransaction(network, m_envInfo, m_statePre, m_transactions[i].transaction);
```

Comment:
You should try to improve `executeTransaction()` function first.
1. Do you need preState after this function. If not do not copy the State.
2. Can it also return changelog if you need it?
3. Use `std::tie()` to unpack tuple.